ui/src/shared/components/ResizeContainer.js

```diff
@@ -16,8 +18,8 @@ const ResizeContainer = React.createClass({
 
   getDefaultProps() {
     return {
-      minTopHeight: 200,
-      minBottomHeight: 200,
+      minTopHeight: minimumTopHeight,
```

Comment:
cool. could also name them both `minTopHeight` and spare yourself the explicit assignment via just `minTopHeight,` rather than `minTopHeight: minimumTopHeight,`